servers/rendering/renderer_rd/forward_mobile/render_forward_mobile.cpp

```diff
@@ -68,7 +96,7 @@ void RenderForwardMobile::RenderBufferDataForwardMobile::configure(RID p_color_b
 	RD::DataFormat color_format = RenderForwardMobile::singleton->_render_buffers_get_color_format();
 
 	if (p_msaa == RS::VIEWPORT_MSAA_DISABLED) {
-		if (color_format == RD::DATA_FORMAT_B10G11R11_UFLOAT_PACK32) {
+		if (color_format == RD::DATA_FORMAT_A2B10G10R10_UNORM_PACK32) {
 			// @TODO add a second color buffer for alpha as this format is RGB only
```

Comment:
I added this as a reminder that the old format didn't support ALPHA, this new one has a 2 bit alpha channel so this check can be removed.